### PR TITLE
POL-243 New Resource Table 5

### PR DESCRIPTION
**Pull request by SheikSena**
### Description

POL-243 New Resource Table 5

### Issues Resolved

[List any existing issues this PR resolves]

### Contribution Check List

- [ ] All tests pass.
- [ ] New functionality includes testing.
- [ ] New functionality has been documented in the README if applicable
- [ ] New functionality has been documented in CHANGELOG.MD

### CODE REVIEW

File: compliance/aws/ecs_unused/aws_unused_ecs_clusters.pt [hunk 1]
```diff
@@ -209,17 +211,24 @@ end
...
+        label "Cluster Name"
+      end
+      field "region" do
+        label "Region"
+      end
+      field "status" do
+        label "Status"
+      end
+      field "Tags" do
+        label "tagKeyValue"
```
Comment (cdwilhelm) [inline compliance/aws/ecs_unused/aws_unused_ecs_clusters.pt]: label should be Tags, field should be tagKeyValue

File: security/aws/elb_unencrypted/aws_elb_encryption.pt [hunk 2]
```diff
@@ -297,17 +302,29 @@ end
...
-    EOS
     escalate $esc_unencrypted_lbs
     check eq(size(data),0)
+    export do
+      resource_level true
+      field "id" do
+        label "LoadBalancer Name"
+      end
+      field "loadBalancerType" do
+        label LoadBalancer Type
```
Comment (cdwilhelm) [inline security/aws/elb_unencrypted/aws_elb_encryption.pt]: label value should be quoted

File: security/aws/elb_unencrypted/aws_elb_encryption.pt [hunk 3]
```diff
@@ -297,17 +302,29 @@ end
...
+        label "Listener Protocol (Port)"
+      end
+      field "region" do
+        label "Region"
+      end
+      field "availabilityZones" do
+        label "Availability Zones"
+      end
+      field "tagKeyValue" do
+        label "TAGs"
```
Comment (cdwilhelm) [inline security/aws/elb_unencrypted/aws_elb_encryption.pt]: Should be Tags

File: security/aws/rds_unencrypted/aws_unencrypted_rds_instances.pt [hunk 4]
```diff
@@ -244,7 +238,37 @@ policy "policy_rds_instance_list" do
...
+        label "DB Engine Name"
+      end
+      field "db_instance_status" do
+        label "DB Instance Status"
+      end
+      field "delete_protection" do
+        label "Delete Protection Enabled"
+      end
+      field "tag_key_value" do
+        label "TAGs"
```
Comment (cdwilhelm) [inline security/aws/rds_unencrypted/aws_unencrypted_rds_instances.pt]: Should be Tags

File: security/aws/rds_unencrypted/aws_unencrypted_rds_instances.pt [hunk 5]
```diff
@@ -227,15 +230,6 @@ end
...
-| RDS Instance Name | Region | Availability Zone | Secondary Availability Zone | Publicly Accessible | DB Engine Name | DB Instance Status | Delete Protection Enabled | TAGs |
-| ----------------- | ------ | ----------------- | --------------------------- | ------------------- | -------------- | ------------------ | ------------------------- | ---- |
-{{ range data -}}
-| {{.db_instance_identifier}} | {{ .region }} | {{ .availability_zone }} |  {{ .sec_availability_zone }} |{{ .publicly_accessible }} | {{ .db_engine_name }} | {{ .db_instance_status }} | {{.delete_protection}} | {{.tag_key_value}} |
-{{ end -}}
-###### Note: Please refer to 'Enable Delete Action' section in the [README](https://github.com/rightscale/policy_templates/tree/master/security/aws/rds_unencrypted) file to perform delete operation.
-    EOS
     escalate $report_unencrypted_RDS_instances
     #Uncomment below five lines (remove '#') to enable Delete action.
     #escalate $delete_unencrypted_RDS_instances_approval
```
Comment (cdwilhelm) [inline security/aws/rds_unencrypted/aws_unencrypted_rds_instances.pt]: In this example the UI will support the actions, so you include **delete** action.  There is no need to include instructions to uncomment.  Readme should be updated to include the action and remove instruction to uncomment etc.

File: security/aws/unencrypted_s3_buckets/aws_unencrypted_s3_buckets.pt [hunk 6]
```diff
@@ -228,25 +228,34 @@ script "js_aws_bucket_unencrypted", type: "javascript" do
...
+        label "Bucket Name"
+      end
+      field "creation_date" do
+        label "Creation Date"
+      end
+      field "region" do
+        label "Region"
+      end
+      field "tagKeyValue" do
+        label "TAGs"
```
Comment (cdwilhelm) [inline security/aws/unencrypted_s3_buckets/aws_unencrypted_s3_buckets.pt]: Should be Tags

File: compliance/aws/ecs_unused/aws_unused_ecs_clusters.pt (hunk 1 above)
Comment (SheikSena) [reply compliance/aws/ecs_unused/aws_unused_ecs_clusters.pt]: Done!

File: security/aws/elb_unencrypted/aws_elb_encryption.pt (hunk 2 above)
Comment (SheikSena) [reply security/aws/elb_unencrypted/aws_elb_encryption.pt]: Done !

File: security/aws/rds_unencrypted/aws_unencrypted_rds_instances.pt (hunk 4 above)
Comment (SheikSena) [reply security/aws/rds_unencrypted/aws_unencrypted_rds_instances.pt]: Done !

File: security/aws/unencrypted_s3_buckets/aws_unencrypted_s3_buckets.pt (hunk 6 above)
Comment (SheikSena) [reply security/aws/unencrypted_s3_buckets/aws_unencrypted_s3_buckets.pt]: Done !

File: security/aws/rds_unencrypted/aws_unencrypted_rds_instances.pt (hunk 5 above)
Comment (SheikSena) [reply security/aws/rds_unencrypted/aws_unencrypted_rds_instances.pt]: Done !

File: cost/aws/unused_rds/unused_rds.pt [hunk 7]
```diff
@@ -288,19 +310,16 @@ end
...
-    label "Escalation approval"
-    description "Approve Unused RDS decommission action"
-    parameter "approval_reason" do
-      type "string"
-      label "Reason for approval"
-      description "Explain why you are approving the action"
-    end
-  end
+  automatic false
+  label "Decommission Unused RDS"
```
Comment (cdwilhelm) [inline cost/aws/unused_rds/unused_rds.pt]: We should standardize on action names.  please change this to Delete RDS Instance, and update the README with the delete changes.

File: security/aws/unencrypted_s3_buckets/aws_unencrypted_s3_buckets.pt [hunk 8]
```diff
@@ -255,32 +264,23 @@ end
...
-    label "Approve Resource Deletion"
-    description "Approve escalation to run RightScale Cloud Workflow to delete the Unencrypted S3 Buckets"
-    parameter "approval_reason" do
-      type "string"
-      label "Reason for Approval"
-      description "Unencrypted S3 Buckets will be deleted After Approval"
-    end
-  end
+  automatic false
+  label "Delete Unencrypted S3 Buckets"
```
Comment (cdwilhelm) [inline security/aws/unencrypted_s3_buckets/aws_unencrypted_s3_buckets.pt]: Change to **Delete S3 Bucket**

File: security/aws/unencrypted_s3_buckets/aws_unencrypted_s3_buckets.pt [hunk 9]
```diff
@@ -255,32 +264,23 @@ end
...
-    label "Approve S3 Buckets Encryption Config Change"
-    description "Approve escalation to run RightScale Cloud Workflow to update encryption for the Unencrypted S3 Buckets"
-    parameter "approval_reason" do
-      type "string"
-      label "Reason for Approval"
-      description "Default encryption will be enabled(AES-256) for the Unencrypted S3 Buckets After Approval"
-    end
-  end
+  automatic false
+  label "Approve S3 Buckets Encryption Config Change"
```
Comment (cdwilhelm) [inline security/aws/unencrypted_s3_buckets/aws_unencrypted_s3_buckets.pt]: Change to **Update S3 Encryption**

**Comment by cdwilhelm** [comment]
@SheikSena can you please fix the conflicting files.  looks like a minor change on the changelogs.  thx

**Pull request merged**